src/rust/aggregator/sqlx_queries/order_history_pipelines/get_prices.sql

```diff
@@ -0,0 +1,11 @@
+WITH parameters AS (
+    SELECT
+        $1::NUMERIC(20,0) AS "market_id",
+        $2::NUMERIC(20,0) AS "txn_version"
+)
+SELECT price
+FROM fill_events, parameters
+WHERE fill_events.market_id = parameters.market_id
+AND fill_events.txn_version < parameters.txn_version
```

Comment:
How about `<=`, since multiple trades can happen in a single txn (composability)

Should probably also do by DESC on event_idx if the field is available